### PR TITLE
Bear_Installation_Tool.rst: Clarify cib's presence

### PR DESCRIPTION
**Pull request by supergr35**
clarify that cib is broken

Add a sentence explaining that cib
is broken

Closes https://github.com/coala/coala/issues/4344

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
GitCommitBear has different error messages for each type of problem.

File: docs/Developers/Bear_Installation_Tool.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
 Bear Installation Tool
 ======================
 
+**PLEASE NOTE: cib is currently broken. Most of the commands listed here will
+ not work.**
```
Comment (SanketDG) [inline docs/Developers/Bear_Installation_Tool.rst]: can we instead have one of those sphinx/docutils directives like a note/warning?

the syntax is something like `.. warning:` but I am not fully sure

**Comment by jayvdb** [comment]
ack b49b90b

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**